src/app/shared/location-riddle-post/location-riddle-post.component.html

```diff
@@ -1,104 +1,106 @@
-				<p style="padding-left: 0.3rem">add a comment...</p>
-			</ion-item>
-			<ion-fab horizontal="start" slot="fixed" vertical="bottom">
-				<ion-fab-button (click)="locationRiddleState.toggleMap.next()">
-					<ion-icon name="{{ locationRiddleState.showMap() ? 'image' : 'map' }}-outline"></ion-icon>
-				</ion-fab-button>
-			</ion-fab>
-			@if (locationRiddleState.marker() && !locationRiddleState.submitted()) {
-				<ion-fab horizontal="end" slot="fixed" vertical="bottom">
-					<ion-fab-button color="success" (click)="submit()">
-						<ion-icon color="dark" name="checkmark-outline"></ion-icon>
-					</ion-fab-button>
-				</ion-fab>
-			}
-		</ion-card-content>
-	</ion-card>
+  <ion-card style="width: 90vw; min-height: 37rem; max-width: 40rem" color="light">
+    <ion-card-header class="ion-padding">
+      <ion-card-title>
+        <ion-item lines="none">
+          <ion-avatar aria-hidden="true" slot="start">
+            <ion-img src="https://ionicframework.com/docs/img/demos/avatar.svg"></ion-img>
+          </ion-avatar>
+          <div style="display: flex; justify-content: space-between; flex-grow: 1">
+            <ion-label class="hover-pointer" color="dark"
+                       (click)="router.navigate(['/home/profiles/' + locationRiddleState.locationRiddle()!.username])">
+              <h2>{{ locationRiddleState.locationRiddle()!.username }}</h2>
+              <p>{{ locationRiddleState.locationRiddle()!.createdAt | date: 'short' }}</p>
+            </ion-label>
+            <app-rating
+              [rating]="locationRiddleState.locationRiddle()!.rating"
+              (makeRating)="locationRiddleState.rateLocationRiddle.next($event)"
+              [ratingError]="ratingError"
+            ></app-rating>
+          </div>
+        </ion-item>
+      </ion-card-title>
+    </ion-card-header>
+    <ion-card-content style="height: 40vh; min-height: 30rem">
+      @if (locationRiddleState.showMap()) {
+        <app-map
+          (placeMarker)="locationRiddleState.placeGuess.next($event)"
+          [marker]="locationRiddleState.marker()"
+          [guesses]="locationRiddleState.guesses()"
+          [userGuess]="locationRiddleState.userGuess()"
+          [solution]="locationRiddleState.solution()"
+          [solved]="locationRiddleState.locationRiddle()!.solved"
+          [zoom]="locationRiddleState.mapZoom()"
+          [center]="locationRiddleState.mapCenter()"
+          (zoomChange)="locationRiddleState.mapZoomChanged.next($event)"
+          (centerChange)="locationRiddleState.mapCenterChanged.next($event)"
+        ></app-map>
+      } @else {
+        <ion-img
+          src="data:image;base64, {{ locationRiddleState.locationRiddle()!.locationRiddleImage }}"
+        ></ion-img>
+      }
+      <ion-item
+        expand="block"
+        class="ion-no-padding"
+        lines="none"
+        style="margin-top: 15px; margin-bottom: 5px"
+        (click)="commentsModalOpen = true"
```

Comment:
formatting changes... but I introduced the class variable commentsModalOpen that is set to true on click of the comments button, the modal now uses [isOpen] instead of trigger, this fixed the issue for me